include/Core/EmailLog.php

```diff
@@ -65,6 +65,14 @@ class EmailLog {
 	public $table_manager;
 
 	/**
+	 * Email Log Plugins Setting.
+	 *
+	 * @since 2.1.0
+	 * @var \EmailLog\Core\UI\Setting\Setting
+	 */
+	public $setting;
```

Comment:
Do we need to reference this variable anywhere? If no, then set this as an `loadie`

Reply:
I'm sorry, I didn't get this. Should this variable be moved to interface `loadie`?

Reply:
What I meant is to remove the variable and then pass a reference to the settings object using the `add_loadie` function.

That way we can remove a member variable in the class and make it much more decoupled.

Reply:
> What I meant is to remove the variable and then pass a reference to the settings object using the `add_loadie` function.

Perfect! Just understood.

Reply:
> That way we can remove a member variable in the class and make it much more decoupled.

Perfect, just one line change, but gives an entirely different meaning. Fixed it now.